client/packages/host/src/components/Login/LoginLayout.tsx

```diff
@@ -83,28 +85,37 @@ export const LoginLayout = ({
       </Box>
       <Box
         flex="1 0 50%"
+        flexDirection="column"
+        alignItems="center"
+        display="flex"
         sx={{
           backgroundColor: 'background.login',
-          alignItems: 'center',
-          justifyContent: 'center',
           overflowY: 'scroll',
         }}
-        display="flex"
       >
-        <Box style={{ width: 285 }}>
-          <form onSubmit={onLogin} onKeyDown={handleKeyDown}>
-            <Stack spacing={5}>
-              <Box display="flex" justifyContent="center">
-                <LoginIcon />
-              </Box>
-              {UsernameInput}
-              {PasswordInput}
-              {ErrorMessage}
-              <Box display="flex" justifyContent="flex-end">
-                {LoginButton}
-              </Box>
-            </Stack>
-          </form>
+        {ServerInfo}
```

Comment:
feels nit-picky to mention this.. but I don't like how it looks up the top of the screen like that. Changes the whole feel of the login screen.

Actually, I don't think the server adds anything here and there's no need to display it.

Reply:
Another area where I just wanted to show that from non bundled (from served) front end, we can access client. No preference on keeping or removing it